cluster/service/cluster.go

```diff
@@ -145,3 +147,39 @@ func (c clusterService) InitializeClusterWatcher() (func() error, error) {
 
 	return watcher.Close, err
 }
+
+// CreateIdentityCluster populate Identity Cluster relationship
+func (c clusterService) CreateIdentityCluster(ctx context.Context, identityID, clusterURL string) error {
```

Comment:
should we name this function `LinkClusterToIdentity` (or `LinkIdentityToCluster`), maybe?

Reply:
sure